### DISCUSSION TITLE
I can not load my page on other devices than my own

### DISCUSSION THREAD

**Discussion by JaisJessen** [community/community#22839] [Other Feature Feedback, Questions, & Ideas]
<p>I am a new user on github, and have experienced this problem:</p>
<p>When i share my public page (<a href="https://jaisjessen.github.io/fulli/" rel="noopener nofollow ugc">https://jaisjessen.github.io/fulli/</a>), it is not visible, but I can easily acccess it on my own laptop.</p>
<p>Is there anything I could’ve done wrong, with the setup?</p>
<p>Any input is appreciated - thanks a lot!</p>
<p>/Jais</p>

**Comment by otter-computer** [comment, marked as answer]
<p>Hi <a class="mention" href="/jaisjessen">@JaisJessen</a>, welcome to the GitHub Support Community!</p>
<p>Looking at the source code for your site it looks like you’re referencing assets using the external URL <code>https://fulli.local/</code>:</p>
<pre><code class="lang-auto">&lt;script src="https://fulli.local/wp-includes/js/wp-emoji-release.min.js?ver=5.5.1" type="text/javascript" defer=""&gt;&lt;/script&gt;
</code></pre>
<p>This may be working on your local machine as you’ve set up this URL using your <code>hosts</code> file to point to a local development server or, if these files are available at that URL on the internet, access has been limited to your specific IP.</p>
<p>You’ll need to update these asset links to point to files from inside your repository, either using relative links or pointing to links that use your <code>github.io</code> URL, or to an external URL where these files are available.</p>

**Comment by JaisJessen** [comment]
<p>Okay, that sounds probable, and makes sense. I am quite the rookie, so where would I change the source code; is that in a hmtl, js og css file?</p>
<p>Thanks for the help, and thanks in advance!</p>
<p>Jais Jessen</p>

**Comment by Bella-01** [comment]
<aside class="onebox allowlistedgeneric">
  <header class="source">
      <a href="https://bella-01.github.io/coursera-test/" target="_blank" rel="noopener nofollow ugc">coursera-test</a>
  </header>
  <article class="onebox-body">
    <img src="" class="thumbnail" width="" height="">

<h3><a href="https://bella-01.github.io/coursera-test/" target="_blank" rel="noopener nofollow ugc">Welcome to GitHub Pages</a></h3>

<p>Coursera test repository</p>


  </article>
  <div class="onebox-metadata">
    
    
  </div>
  <div style="clear: both"></div>
</aside>
<br>
this is my own link,still showing 404. i can’t view my webpage.
